addons/website_sale/static/src/js/website_sale_form_editor.js

```diff
@@ -8,19 +8,23 @@ FormEditorRegistry.add('create_customer', {
         type: 'char',
         modelRequired: true,
         name: 'name',
+        fillWith: 'name',
```

Comment:
Commit title is `[IMP] website : add pre fill option for website form` but it is also modifying other modules than website, needs to be indacted too.
Also remove the whitespace between `website :`